src/components/WeaponComponent.vue

```diff
@@ -156,21 +174,21 @@ export default {
 
 	.name {
 		$padding: 25px;
+		$list-padding: 10px;
```

Comment:
Could have a bit more padding here still 😄 
```suggestion
		$list-padding: 20px 10px;
```